src/continuum_absorption/absorption_ff_metal.jl

```diff
@@ -0,0 +1,458 @@
+using Interpolations: LinearInterpolation, Throw
+
+# We have implemented the free-free metal departure terms (from Peach 1970) for the same species as
+# MARCS/Turbospectrum (see Table 1 of Gustafsson+ 2008).
```

Comment:
We should enumerate these somewhere.  It's
 - He I
 - C I
 - C II
 - Mg I
 - Si I
 
Right?